NAMESPACE

```diff
@@ -1,13 +1,15 @@
 # Generated by roxygen2: do not edit by hand
 
 S3method("[",pmx_gpar)
+S3method(base::print,classIntervals)
```

Comment:
Should this be exported?  I think it shouldn't be exported

Reply:
Actually it likely should be exported, but the "classIntervals" `s3` should be changed since `classInt` is  a popular package and is likely to cause issues with many other packages.